css-fonts/styles.css

```diff
@@ -0,0 +1,4 @@
+body {
+  font-family: 'M PLUS 1 Code', sans-serif;
+  line-height: 2;
+}
```

Comment:
Try using two different fonts, one for the headings and one for the paragraphs:

---

![image](https://user-images.githubusercontent.com/16143771/144310872-e6f16472-4897-4462-8db1-b6da757c5602.png)